requirements/base_merlin.txt

```diff
@@ -1 +1 @@
-merlin-models>=0.11
+merlin-models>=23.3.0
```

Comment:
shall we update it to 23.4.0?

Reply:
Yes, we'll need to update this to 23.4.0 once the models package is published

Reply:
updated